distribution/docs/src/main/resources/_contents/_solr-contents/managing-solr-contents.adoc

```diff
@@ -309,3 +309,50 @@ Here are some helpful suggestions for setting up data backups and recoveries:
 * Run a scheduler program that calls the backup URL on a timed basis.
 * Put indicators in place that can detect when data corruption may have occurred.
 * Testing a backup before recovery is possible. A replicated "staging" Solr server instance can be stood up, and the backup can be copied to that system for testing before moving it to the "production" system.
+
+=== High Availability (Optional)
+
+==== SolrCloud
+
+SolrCloud provides distributed capabilities of a highly available, fault tolerant cluster of Solr servers.
+
```

Comment:
If I'm reading the SolrCloud documentation correctly, it seems that it is a configurable feature on the standard SOLR server that allows instances to communicate with each other in a highly-available fashion. We should revise the above line to be explicitly clear. I do not believe SolrCloud is, itself, the cluster of servers. 

Perhaps: 
```
SolrCloud provides the distributed capabilities needed by SOLR servers to communicate and establish a highly available, fault tolerant cluster of SOLR instances. 
```

I think this needs to be discussed with the team.